api/graphql/directives/protected.js

```diff
@@ -14,6 +14,19 @@ export default class ProtetedDirective extends SchemaDirectiveVisitor {
         field._parentEntity ||
         camelcase(info.parentType.name);
 
+      if (_this.args.post_execution) {
+        const result = await resolve.call(this, parent, _args, ctx, info);
+
+        if (entityName === 'query') parent = entityName;
```

Comment:
Why have you done this?

Reply:
I don't think entity is required in the auth policy, no need to override parent here.